tests/python-pytest/onnx/test_operators.py

```diff
@@ -927,3 +927,15 @@ def test_onnx_export_convolution(tmp_path, dtype, shape, num_filter, num_group,
                   **kwargs)
     inputs = [x, w] if no_bias else [x, w, b]
     op_export_test('convolution', M, inputs, tmp_path)
+
+
+@pytest.mark.parametrize('dtype', ['int32', 'int64', 'float16', 'float32', 'float64'])
+@pytest.mark.parametrize('axis', [-3, -2, -1, 0, 1, 2])
+@pytest.mark.parametrize('mode', ['clip', 'wrap'])
```

Comment:
Shall we also test raise mode/

Reply:
The raise mode will raise an error when indices out of bound. Do you have suggestions how we can test it?

Reply:
We can split it into multiple tests, one test just for 'raise' mode and the other tests the 'clip' and 'wrap' modes, since the parametrized options will be different. 

Sorta like this:
https://github.com/apache/incubator-mxnet/pull/19855/files#diff-30a9f0d694eb3951b810ec760f714b3873ab4453e685620b1571991e644aae3cR859-R881